grasp_apps/draw_x/launch/draw_x.yaml

```diff
@@ -0,0 +1,5 @@
+ur_test:
```

Comment:
can you check why draw_x app rely on the ur_test?

Reply:
This should be the node name of the draw_x. I forgot to correct it.